avm/res/web/serverfarm/main.bicep

```
@@ -18,7 +18,7 @@ param name string
 })
 param skuName string = 'P1v3'
 
-@description('Required. Number of workers associated with the App Service Plan.')
+@description('Optional. Number of workers associated with the App Service Plan. This defaults to 3, to leverage availability zones..')
```

Minor typo, please bare with me :) 
```suggestion
@description('Optional. Number of workers associated with the App Service Plan. This defaults to 3, to leverage availability zones.')
```

---

All done. Thanks @eriqua, love the eagle eyes!